lib/2fa.coffee

```diff
@@ -47,8 +47,14 @@ get2fa = (deps, opts) ->
 	# });
 	###
 	exports.isEnabled = (callback) ->
-		token.getProperty('twoFactorRequired').then (twoFactorRequired) ->
-			return twoFactorRequired?
+		auth.getType().then (tokenType) ->
+			if tokenType != 0
```

Comment:
Typically this should really use the enum type in resin-auth, rather than depending on the specific magic number (which willl change, if somebody reorders these two lines: https://github.com/resin-io-modules/resin-auth/blob/master/lib/token.ts#L31-L32). That'd also let you avoid the comment below, and make this more readable.

In fact though, I don't think you need this check at all. `needs2FA` is implemented for API keys too, but just returns false, so without this check I think this code will still do the same thing: https://github.com/resin-io-modules/resin-auth/blob/7a692b306174af5f89cd6979ecc85463f578f576/lib/api-key.ts#L102